profiles/common/modules/features/nexteuropa_newsroom/theme/related-content.tpl.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
I simplified the template, but still don't think we should move the logic which just displays or hides HTML markup